apis/database/v1beta1/nosql_types.go

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
```

Comment:
This code looks like it's at or close to what we'd consider v1beta1 quality, but per https://github.com/crossplane/provider-gcp/pull/212#discussion_r408547494 we recently decided we'd prefer to introduce all new API types at v1alpha1 to let them soak a little before we promote them to v1beta1. Would you mind switching this to v1alpha1?

Reply:
v1alpha1 does not work because auto-generation fails due to a namespace conflict. I am using v1alpha3 instead.

$ make generate
13:05:36 [ .. ] go generate darwin_amd64
angryjet: error: error loading packages using pattern ./...: /Users/asouza/code/crossplane/provider-azure/apis/database/v1alpha1/zz_generated.deepcopy.go:32:23: cannot use &out.Spec (value of type *CosmosDBAccountSpec) as *v1alpha1.ResourceSpec value in argument to in.Spec.DeepCopyInto
exit status 1